assets/sass/components/dashboard/_googlesitekit-subtle-notification.scss

```diff
@@ -26,33 +26,42 @@
 	padding: $grid-gap-phone;
 
 	@media (min-width: $bp-tablet) {
+		align-items: center;
 		flex-direction: row;
 		gap: $grid-gap-phone;
 		padding: 18px $grid-gap-desktop;
 	}
-}
 
-.googlesitekit-subtle-notification__icon {
-	display: none;
+	.googlesitekit-subtle-notification__icon {
+		display: none;
 
-	@media (min-width: $bp-tablet) {
-		display: block;
+		@media (min-width: $bp-tablet) {
+			display: block;
+		}
 	}
-}
 
-.googlesitekit-subtle-notification__content {
-	color: $c-content-on-primary-container;
-	font-family: $f-primary;
-	font-size: $fs-title-md;
-	font-weight: $fw-medium;
-	line-height: $lh-title-lg;
+	.googlesitekit-subtle-notification__content {
+		color: $c-content-on-primary-container;
+		font-family: $f-primary;
+		font-size: $fs-title-md;
+		font-weight: $fw-medium;
+		line-height: $lh-title-lg;
 
-	@media (min-width: $bp-tablet) {
-		flex: 1;
-		font-size: $fs-title-lg;
+		@media (min-width: $bp-tablet) {
+			flex: 1;
+			font-size: $fs-title-lg;
+		}
+
+		p {
+			@media (min-width: $bp-tablet) {
+				margin: 0;
+			}
+		}
 	}
 
-	p {
+	.mdc-button {
+		margin: 0 auto;
```

Comment:
I moved other styles under parent selector. Basically only change here is this line